changelog/7305.feature.rst

```diff
@@ -0,0 +1 @@
+New `required_plugins` configuration option allows the user to specify a list of plugins required for pytest to run. An error is raised if any required plugins are not found when running pytest.
```

Comment:
I think RST requires double ticks though I'm never sure TBH...

```suggestion
New ``required_plugins`` configuration option allows the user to specify a list of plugins required for pytest to run. An error is raised if any required plugins are not found when running pytest.
```

Reply:
It does I’ll make the change. Thank you for catching